apps/src/templates/progress/ProgressBubble.jsx

```diff
@@ -41,7 +41,6 @@ class ProgressBubble extends React.Component {
     lessonTrophyEnabled: PropTypes.bool,
     pairingIconEnabled: PropTypes.bool,
     hideToolTips: PropTypes.bool,
-    stageExtrasEnabled: PropTypes.bool,
```

Comment:
This was not being used at all so removed it

Reply:
When peeking into this file I noticed there is also an unused style called `disabledStageExtras`. Mind deleting that too?